api

```diff
@@ -557,6 +557,18 @@ list_subtract() { #Outputs a list of apps from stdin, minus the ones that appear
 }
 
 read_category_files() { #Generates a combined categories-list from several sources: category-overrides, global categories file, and unlisted apps. Format: "app|category"
+  
+  #remove app category if app folder not found
+  IFS=$'\n'
+  for app in $(cat "${DIRECTORY}/data/category-overrides"); do 
+  	if ! [ -d "${DIRECTORY}/apps/$(echo "$app" | sed 's/|.*//')" ] &>/dev/null; then 
```

Comment:
Can we switch out these tab characters for double-spaces to be consistent with the rest of the scripts?

Reply:
Fixed.